Lib/uu.py

```diff
@@ -134,7 +134,7 @@ def decode(in_file, out_file=None, mode=None, quiet=False):
         elif isinstance(out_file, str):
             fp = open(out_file, 'wb')
             try:
-                os.path.chmod(out_file, mode)
+                os.chmod(out_file, mode)
             except AttributeError:
```

Comment:
Perhaps I'm missing something but `os.chmod()` is defined undonditionally in `Modules/posixmodule.c`:

https://github.com/python/cpython/blob/941ec210aaef517cf818b043ec28345962f78465/Modules/posixmodule.c#L2821

Do we still need the `try...except AttributeError` check? It was added in 1995 in https://github.com/python/cpython/commit/8b745126d5e550c366e949dbb5c9294ec6074c7c#diff-bd8ce21226a634ebc2701b92641e3424R108 and I think we can remove it now.

Reply:
Agree. I've removed the `try ... except` block.